modules/rosa-sts-associating-your-aws-account.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-quickly.adoc
+
+:_content-type: PROCEDURE
+[id="rosa-sts-associating-your-aws-account_{context}"]
+= Associating your AWS account with your Red Hat organization
+
+Before using {cluster-manager-first} to create {product-title} (ROSA) clusters that use the AWS Security Token Service (STS), create an {cluster-manager} IAM role and link it to your Red Hat organization. Then, create a user IAM role and link it to your Red Hat user account in the same Red Hat organization.
+
+.Prerequisites
+
+* You have completed the AWS prerequisites for ROSA with STS.
+* You have available AWS service quotas.
+* You have enabled the ROSA service in the AWS Console.
+* You have installed and configured the latest ROSA CLI (`rosa`) on your installation host.
++
+[NOTE]
+====
+To successfully install ROSA clusters, use the latest version of the ROSA CLI.
+====
+* You have logged in to your Red Hat account by using the `rosa` CLI.
+
+.Procedure
+
+. Create an {cluster-manager} role and link it to your Red Hat organization:
++
+[source,terminal]
+----
+$ rosa create ocm-role --admin
```

Comment:
I think it will be helpful if we can give some reference on what is the difference between admin and not admin ocm role here. like adding a link to the section.

Reply:
Thank you for your suggestion. I have subsequently added the following note under this step:

****Note**: If you want to enable automatic deployment of the cluster-specific Operator roles and the OpenID Connect (OIDC) provider using {cluster-manager}, you must apply the administrative privileges to the role. For more information about the basic and administrative privileges for the {cluster-manager} role, see _Understanding AWS account association_.**